pkg/yang/entry.go

```diff
@@ -660,6 +660,22 @@ func ToEntry(n Node) (e *Entry) {
 		// when the group is used in multiple locations and the
 		// grouping has a leafref that references outside the group.
 		e = ToEntry(g).dup()
+
+		switch determineYangVersion(g) {
```

Comment:
We haven't added this kind of error checking elsewhere -- I wonder whether we should try and include the parsed YANG version somewhere in the output entry. WDYT?

Reply:
if we want to be strict then thats probably a good idea.